pkg/lint/lintersdb/manager.go

```diff
@@ -488,6 +490,11 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 			WithLoadForGoAnalysis().
 			WithURL("https://github.com/Djarvur/go-err113"),
 
+		linter.NewConfig(golinters.NewGoFactoryLint(goFactoryCfg)).
+			WithSince("next_version").
```

Comment:
Which version should I set?

Reply:
https://github.com/golangci/golangci-lint/pull/4196#issuecomment-1807228984